src/cswinrt/code_writers.h

```diff
@@ -5383,6 +5393,22 @@ private % AsInternal(InterfaceTag<%> _) => _default;
             default_interface_name,
             default_interface_name,
             bind<write_lazy_interface_initialization>(type),
+            [&](writer& w)
```

Comment:
Do we care about doing the same in the default constructor (the one a C# caller would new up)?  And how does GCPressureAttribute apply to derived types?  Is it possibly for the base type to just have it or will any derived types also have that attribute?  If not, maybe we want it in the protected derived constructor too?